src/main/java/cafeboard/Post/PostResponse.java

```diff
@@ -0,0 +1,13 @@
+package cafeboard.Post;
+
+public record PostResponse(
+        Long id,
+        String title,
+        int commentCount
+
+)
+
+{
+
+
+}
```

Comment:
Ctrl + Alt + L 활용해서 줄 맞추고 필요 없는 공백을 지우는 것도 사소하지만 은근 중요합니다.
```suggestion
public record PostResponse(
        Long id,
        String title,
        int commentCount
) {
}
```